src/app/progress/character-tile.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import { IDestinyCharacterComponent } from '../bungie-api/interfaces';
+
+interface CharacterTileProps {
+  character: IDestinyCharacterComponent;
+  defs;
+  lastPlayedDate: Date;
+}
+
+export function CharacterTile(props: CharacterTileProps) {
+  const { defs, character, lastPlayedDate } = props;
+
+  const race = defs.Race[character.raceHash];
+  const gender = defs.Gender[character.genderHash];
+  const classy = defs.Class[character.classHash];
+  const genderRace = race.genderedRaceNames[gender.genderType === 1 ? 'Female' : 'Male'];
+  const className = classy.genderedClassNames[gender.genderType === 1 ? 'Female' : 'Male'];
+  const current = lastPlayedDate.getTime() == new Date(character.dateLastPlayed).getTime() ? 'current' : '';
+
+  // TODO: update this to be a D2-specific, simplified tile
+  return <div className={`character ${current ? 'current' : ''}`}>
```

Comment:
I'm actually surprised this is valid syntax... When returning multiple lines of JSX you should wrap it in parens

```
return (
  <div ...
  </div>
)
```

Reply:
Oh, good tip. It seems to work OK like this, but I'll keep that in mind.